contrib/win32/openssh/OpenSSHBuildHelper.psm1

```diff
@@ -201,8 +213,13 @@ function Start-OpenSSHBootstrap
     # Update machine environment path
     if ($newMachineEnvironmentPath -ne $machinePath)
     {
+        if (-not (IsAdministrator))
```

Comment:
We don't need administrative privileges to build openssh when you have all the prerequisites. 
Having this change will always require the administrative console window.